repos/keg-core/core/base/assets/index.js

```diff
@@ -4,5 +4,5 @@
 // ********************************* WARNING *********************************
 
 export const assets = {
-  ['clientLogo']: require('/home/runner/work/keg-hub/keg-hub/repos/keg-core/core/base/assets/clientLogo.jpg')
+  ['clientLogo']: require('/Users/michael.carolin/keg-hub/repos/keg-core/core/base/assets/clientLogo.jpg')
```

Comment:
This file should really be ignored in git, It seems odd this change was picked up.
That said, it seems you were running your changes on local, which is why it updated to your home path. 

Can you revert this file from the PR. Thanks

Reply:
Updated